.github/workflows/integration.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
         with:
-          go-version: '1.21'
+          go-version: '1.22'
```

Comment:
We can leverage `go-version-file` (if the `go.mod` file is present, ie: the repo is cloned)